README.md

```diff
@@ -41,6 +41,7 @@ The file ReadMe.txt in a build bundle describes how to install the library on th
    16. [Cross Platform Build and Test on Linux for MinGW](#2-16)
    17. [Building with C++11 and other compilers](#2-17)
    18. [Static and Shared Libraries](#2-18)
+   19. [Support for bmff files (CR3, HIF, AVIF and HEIC)](#2-19)
```

Comment:
I'd stick to "HEIF" in the documentation instead of proliferating "HIF" - it's just an extension Canon chose (poorly), but it really is just an [ordinary HEIF/HEIC file](https://github.com/strukturag/libheif/issues/219).